### DISCUSSION TITLE
Float v double

### DISCUSSION THREAD

**Discussion by AlexJamesWright** [AlexJamesWright/METHOD#48] [General]
@IanHawke Is there a reason for using floats over doubles in your toy heat model? E.g. https://github.com/AlexJamesWright/METHOD/blob/3d2f53c457995e313b571005799f47406c0bc07c/Project/CPU/Src/toy_q_ce.cc#L48

**Comment by IanHawke** [comment]
No, pure laziness

**Reply by AlexJamesWright** [reply]
Might that not cause problems with precision, convergence test etc?

**Comment by IanHawke** [comment]
Yes; recent commits should shift all (most?) to double. At this stage it
doesn't seem to have significant effects.
